.travis.yml

```diff
@@ -3,7 +3,7 @@ node_js:
   - "10"
 addons:
   code_climate:
-    repo_token: 816f8d801552b918e8d9ad19921ed255a11bddad6089f7a3ae6c6ac783de0091
+    repo_token: $REPO_TOKEN
```

Comment:
env variables usually named just `REPO_TOKEN` or better `CODE_CLIMATE_REPO_TOKEN`

Reply:
I've found out that required name is `CODECLIMATE_REPO_TOKEN`. Now it's set at travis-ci settings.